baybe/recommenders/nonpredictive/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""Nonpredictive recommenders."""
```

Comment:
Yes, definitely. I think we also need some general overhaul of these headers, ideally in a single so that the same style is applied. Right now, it's only to make linters happy and have SOMETHING in the autodoc, instead of an empty cell. But that doesn't mean that I would stop you if you already wanted to extend the text now.

In the long run, I think these headers should have the same logic as class docstrings. That is, one short header line followed by a longer paragraph explaining the content/rational of the module

Reply:
If I would have gotten 1 Euro for everytime you mentiong that something needs a general overhaul or refactoring, I'd be rich by now :D
If we agree that these need to change at some point, I am more than happy to delay this.